src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -44,7 +44,7 @@ interface ReimbursementProductTableProps {
     id: string;
   }[];
   errors: FieldErrors<ReimbursementRequestFormInput>;
-  control: Control<ReimbursementRequestFormInput, any>;
+  control: Control<ReimbursementRequestFormInput, ReimbursementProductFormArgs>;
```

Comment:
you can delete the `, any` and it should work still work. Let's do it that way cause it's a bit simpler

Reply:
done ✅